modules/tide_jira/tide_jira.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Tide Jira
+type: module
+description: Tide Jira integration
+core_version_requirement: ^8.9 || ^9
+package: tide
+dependencies:
+  - jira_rest:jira_rest
```

Comment:
I think this one needs to be added in the dependencies as well - lesstif/php-jira-rest-client. Also the related fields I guess

Reply:
lesstif/php-jira-rest-client is a PHP library, not a module, I don't think it can be specified here?